torchci/lib/bot/autoLabelBot.ts

```diff
@@ -347,17 +346,6 @@ function myBot(app: Probot): void {
     if (!isPyTorchPyTorch(owner, repo)) {
       return;
     }
-    const labels: string[] = context.payload.pull_request.labels.map(
-      (e) => e["name"]
-    );
-    if (labels.find( x => x === CIFLOW_TRUNK_LABEL)) {
-      return;
-    }
-    const reviewer = context.payload.review.user.login;
-    // Ignore reviews from users without write permissions
-    if (!await hasWritePermissions(context, reviewer)) {
-      return;
-    }
```

Comment:
Why did you removed this logic from here? I.e. it was a no-op, but a good one if we plan to implement it later